gtk4/sys/src/lib.rs

```diff
@@ -1114,6 +1120,7 @@ pub const GTK_DEBUG_CSS: GtkDebugFlags = 1048576;
 #[cfg(feature = "v4_18")]
 #[cfg_attr(docsrs, doc(cfg(feature = "v4_18")))]
 pub const GTK_DEBUG_BUILDER: GtkDebugFlags = 2097152;
+pub const GTK_DEBUG_SESSION: GtkDebugFlags = 4194304;
```

Comment:
Similar to the Builder one, this should get a since annot.

Reply:
See https://gitlab.gnome.org/GNOME/gtk/-/merge_requests/9145